### PR TITLE
Look up soak project from --soak flag, rather than project names

### PR DESCRIPTION
**Pull request by krzyzacy**
/assign @yguo0905 @yujuhong

### CODE REVIEW

File: scenarios/kubernetes_janitor.py [hunk 1]
```diff
@@ -105,7 +105,12 @@ def check_ci_jobs():
 
     match_re = re.compile(r'--gcp-project=(.+)')
     for value in config.values():
+        clean_hours = 24
         for arg in value.get('args', []):
+            # lifetime for soak cluster should be 7 days
```
Comment (yguo0905) [inline scenarios/kubernetes_janitor.py]: This is from the old legacy job, which gets redeployed every 7 days?

**Comment by yguo0905** [comment]
/lgtm
Thanks!

File: scenarios/kubernetes_janitor.py (hunk 1 above)
Comment (krzyzacy) [reply scenarios/kubernetes_janitor.py]: that's from https://github.com/kubernetes/test-infra/blob/master/kubetest/main.go#L156, I was trying to keep the same behavior

**Pull request merged**